src/components/Forms/Health/HTTPHealthFormEditor.tsx

```diff
@@ -0,0 +1,94 @@
+import FormikEnvVarConfigsFields from "../Formik/FormikConfigEnvVarFields";
+import FormikConfigFormFieldsArray from "../Formik/FormikConfigFormFieldsArray";
+import FormikTextInput from "../Formik/FormikTextInput";
+import FormikCheckbox from "../Formik/FormikCheckbox";
+import FormikTemplateFields from "../Formik/FormikTemplateFields";
+import FormikJSONCheckFields from "../Formik/FormikJSONCheckFields";
+import FormikCheckboxFieldsGroup from "../Formik/FormikCheckboxFieldsGroup";
+
+type HTTPHealthFormEditorProps = {
+  fieldName: string;
+};
+
+export function HTTPHealthFormEditor({ fieldName }: HTTPHealthFormEditorProps) {
+  return (
+    <>
+      <FormikTextInput name={`${fieldName}.name`} label="Name" required />
+      <FormikTextInput name={`${fieldName}.description`} label="Description" />
+      <FormikTextInput name={`${fieldName}.icon`} label="Icon" />
+      {/* add labels here */}
+      <FormikTemplateFields name={`${fieldName}.test`} label="Test" />
```

Comment:
Create form groups /  sections:

Name / Icon.  (one line)
Description
URL (alias to endpoint)
Method: GET (default) | PUT | POST |  HEAD (button group
Body: (If method ==  PUT/POST), textarea
Authentication:
Headers:

**Test** 
`HTTP Response Code`: (Use multi react-select component (Call it MultiSelectArrayField or similar) with a list of : 200,201,202,204,301,302,401,404,500,502,503) + the option to type in a new  value
`Max Response Time (in millis)`:
`Response Text`: 
*Exact response content expected to be returned by the endpoint*
`Max SSL Expiry Age (days)`: 
*Maximum number of days until the SSL Certificate expires.*
Script: `None | Go Template | Javascript | JSONPath | Expr` (when select a 3 line editor is shown below)

**Advanced**
`Customize Response`:  `None | Go Template | Javascript | JSONPath | Expr`  (display field)
`Transform Response`:  `None | Go Template | Javascript | JSONPath | Expr`  (display field)